src/audit-license-headers.js

```diff
@@ -30,21 +30,21 @@ var repoutil = require('./repoutil');
 // constants
 var COMMON_RAT_EXCLUDES = [
     // Binary Files
-    '*.wav',
-    '*.webloc',
+    '(.*).wav',
+    '(.*).webloc',
 
     // iOS Related
-    '*.xcworkspacedata',
-    '*.xccheckout',
-    '*.xcscheme',
-    '*.xcodeproj',
-    '*-Info.plist',
+    '(.*).xcworkspacedata',
+    '(.*).xccheckout',
+    '(.*).xcscheme',
+    '(.*).xcodeproj',
+    '(.*)-Info.plist',
     'Info.plist',
 
     // Other
-    '.*',
-    '*.json', // Excludes all JSON files because commenting is not supported.
-    '*.txt', // Excludes all text files because commenting is not supported.
```

Comment:
did this get lost intentionally?

Reply:
Ah yes, the `txt` was lost. I am adding back now.